index.html

```diff
@@ -1199,6 +1228,35 @@ <h3>Multistatus Correlation</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Status Message Alterations</h3>
+
+      <p>
+When a status list uses the status messages feature, it becomes possible for
+the <a>issuer</a> to increase the type of messages that are associated with
+the <a>verifiable credentials</a> it issues over time.
+      </p>
+
+      <p>
+This feature creates a potential privacy violation where <a>holder</a> of the
+<a>verifiable credential</a> might be associated with additional status
+information that was not present when the original <a>verifiable credential</a>
+was issued. For example, initial status messages might convey "delayed" and
+"canceled", but additional status messages might be added by the <a>issuer</a>
+to convey "delayed due to non-payment" and "canceled due to illegal activity",
+which would not be apparent to the <a>holder</a> unless there was monitoring
+software operating on behalf of the holder that would warn them that the
+<a>issuer</a> intends to expose additional information about their activity.
+      </p>
+
+      <p>
+Holder software can provide features to <a>holders</a> that warn them about
+their level of information exposure when using <a>verifiable credentials</a>
+that are associated with status messages and warn them when the level of
+information exposure changes.
+      </p>
```

Comment:
This paragraph is unclear, since it not clear who the status messages are about.

Reply:
@TallTed clarified the language in some of his change suggestions, which have now been applied. The status messages are about VCs, which express information about subjects/holders.

Reply:
In which way do VCs express information about holders? (when they are not subjects)

Reply:
@David-Chadwick —

> [it not clear who the status messages are about](https://github.com/w3c/vc-bitstring-status-list/pull/156/files#r1537357327)

Status messages are always about VCs/VPs; in other words, your question should be "what", not "who". In this sentence, the status messages are about `<a>verifiable credentials</a> that are associated with status messages`.

> [In which way do VCs express information about holders? (when they are not subjects)](https://github.com/w3c/vc-bitstring-status-list/pull/156/files#r1545490195)

If nowhere else, VCs express information about `holders` when `issuees` are identified, as `issuees` must be `holders`. VCs might also express information about `authorized presenters` who must also be `holders`, as they cannot present something they don't hold.

Reply:
Privacy issues can only be about living people. They are not about verifiable credentials, but about the subjects of them (or the issuee if this not-yet-accepted definition is accepted and the property is added to the VC, which no-one is currently proposing). Hence my original question.

Reply:
@David-Chadwick — I think your original question has been addressed by subsequent changes to the text, which now reads:
```
Holder software can provide features to <a>holders</a> that warn them about the
level of <a>holder</a> and/or <a>subject</a> information exposure when using
<a>verifiable credentials</a> that are associated with status messages, and warn
them when the level of information exposure changes.
```
To rephrase, `<a>verifiable credentials</a> that are associated with status messages` expose some `level of <a>holder</a> and/or <a>subject</a> information`, about which level, and changes to that level, `holder software can warn <a>holders</a>`. I think this rephrasing is more confusing than the current text, but perhaps it will clarify things for you?

Reply:
The text which now reads..... does address my issue. Thankyou. But I cannot understand your re-phrasing paragraph below it. Sorry. I agree with you that it is very confusing!!